docs-src/concepts/what-is-a-workflow-definition.md

```diff
@@ -8,15 +8,136 @@ tags:
   - explanation
 ---
 
-A Workflow Definition is the code that defines the constraints of a Workflow Execution.
+A Workflow Definition is the code that defines the Workflow and written with a programming language and a corresponding Temporal SDK.
+A Workflow Definition defines the constraints of a Workflow Execution.
+
+Below are different ways to develop a basic Workflow Definition.
+
+<Tabs groupId="basic-workflow-definition" queryString>
+<TabItem value="go" label="Go">
+
+**Workflow Definition in Go**
+
+```go
+package yourapp
+
+import (
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+// ...
+
+// YourSimpleWorkflowDefintiion is the most basic Workflow Defintion.
+func YourSimpleWorkflowDefinition(ctx workflow.Context) error {
+	// ...
+	return nil
+}
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+**Workflow Definition in Java**
+
+```java
+// Workflow interface
+@WorkflowInterface
+public interface YourWorkflow {
+
+    @WorkflowMethod
+    String yourWFMethod(Arguments args);
+}
+```
```

Comment:
This is just the interface - will probably want to add an example implementation as well.